source/standards/tracking-dependencies.html.md.erb

```diff
@@ -10,43 +10,29 @@ When you develop and operate a service, it’s important to keep any third party
 
 Any automated tools you use to manage third party dependencies should be compatible with [GDS supported programming languages][]. The tools you use should neither slow down your development process nor disclose potential security vulnerabilities to the public.
 
-You can read more about [managing software dependencies in the Service Manual][].
+You can read more about [managing software dependencies in the Service Manual][], where you will find a list of common dependency management tools.
 
-## Programming languages and their dependencies
+Our [programming language style guides][] also contain language-specific advice about managing dependencies (for example, [managing Python dependencies][]).
 
-Depending on what programming language you use, there are differences in how to manage software dependencies. The Service Manual [explains some of these differences][].
+## Monitoring for vulnerabilities
 
-### Ruby
+### Ruby - GOV.UK Gem Security Checker
 
 You should use the [GOV.UK Gem Security Checker][] alongside your regular code checks. This will help your team move any code vulnerabilities into the team’s work backlog. You can then:
 
 * prioritise fixes relative to other project work
 * address vulnerabilities in private before making the fix public
 
-## Dependency management tools
+### Snyk
 
-A common way to manage third party dependencies is to use a dependency management tool. [The Service Manual][] has some suggestions about [how to work with third party code][] and about some common dependency management tools.
+[Snyk][] is a dependency management tool which can [monitor code dependencies][] for security vulnerabilities. Using Snyk allows you to automatically check GitHub pull requests (PRs) for vulnerable dependencies and potentially create a pull request to fix them. It also supports the other languages that GDS uses, including Python, Java and JavaScript.
 
-## Case study - Snyk
+Snyk is being used successfully in the Digital Marketplace and GOV.UK Pay programmes, but configured to ignore vulnerabilities that have no fix, and without automatic generation of pull requests.
```

Comment:
Should we state that we're using both the pre-merge check and the weekly reports? If so I'd reword this to something like:

```
Snyk is being used successfully in the Digital Marketplace and GOV.UK Pay programmes. The Digital Marketplace uses Snyk as a pre-merge check on all pull requests for its public repositories. The developers also review the weekly report for new vulnerabilities, applying fixes manually instead of using Snyk's automatic pull request feature. The reports can be configured to ignore vulnerabilities that have no fix. 
```